src/test/groovy/com/ofg/base/MicroserviceMvcWiremockSpec.groovy

```diff
@@ -1,10 +1,20 @@
 package com.ofg.base
 
 import com.ofg.infrastructure.base.MvcWiremockIntegrationSpec
+import com.ofg.infrastructure.web.correlationid.CorrelationIdFilter
 import com.ofg.twitter.Application
 import org.springframework.boot.test.SpringApplicationContextLoader
 import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.web.servlet.setup.MockMvcBuilders
 
 @ContextConfiguration(classes = [Application], loader = SpringApplicationContextLoader)
 class MicroserviceMvcWiremockSpec extends MvcWiremockIntegrationSpec {
+
+    //TODO: Switch to configureMockMvcBuilder() when micro-infra-spring 0.8.4 is released
+    void setup() {
```

Comment:
Id' rather wait for release. After all it's just a matter of click right now :-).